pac4j-oauth/src/main/java/org/pac4j/oauth/client/TwitterClient.java

```diff
@@ -35,14 +37,14 @@ protected void clientInit() {
         configuration.setApi(getApi());
         configuration.setProfileDefinition(new TwitterProfileDefinition());
         configuration.setHasBeenCancelledFactory(ctx -> {
-            final String denied = ctx.getRequestParameter("denied");
-            if (CommonHelper.isNotBlank(denied)) {
+            final Optional<String> denied = ctx.getRequestParameter("denied");
+            if (CommonHelper.isNotBlank(denied.orElse(null))) {
```

Comment:
prefer using map and/or flatMap and not nulls
and while you are at it: replace those awful return true/false by returning the condition in the if :)